Sources/Private/Experimental/Extensions/CALayer+fillBounds.swift

```diff
@@ -0,0 +1,19 @@
+// Created by Cal Stephens on 12/15/21.
+// Copyright © 2021 Airbnb Inc. All rights reserved.
+
+import QuartzCore
+
+extension CALayer {
+  /// Updates the `bounds` of this layer to fill the bounds of its `superlayer`
+  /// without setting `frame` (which is not permitted if the layer can rotate)
+  func fillBoundsOfSuperlayer() {
```

Comment:
aha right

it would be pretty sick if we could get the linter to do this automatically -- I could imagine this working for either a type-level allowlist (`UIView` / `CALayer` etc) or a prefix-level allowlist (`UI` / `CA` etc)